freetextresponse/private/view.js

```diff
@@ -46,4 +58,42 @@ function FreeTextResponseView(runtime, element) {
         });
         return false;
     });
+
+    buttonSave.on('click', function () {
+        buttonSave.text('Checking...');
```

Comment:
This fine for now and I'm just thinking out loud here, but eventually
we need to start i18n/l10n'ing our XBlocks.

edX has done related work.